### PR TITLE
Feature/add ve ocean delegation

### LINKED ISSUE #1191: Add veOCEAN delegation info

### PR DESCRIPTION
**Pull request by KatunaNorbert**
Fixes #1191.

### CODE REVIEW

**Comment by jamiehewitt15** [comment]
Looks good, I just left a small suggestion

File: rewards/veOcean-Data-Farming-Tutorial.md [hunk 1]
```diff
@@ -76,5 +74,22 @@ All rewards are paid out in OCEAN tokens. On every Thursday the pink "Claim" but
...
+
+If you have multiple wallets, delegation enables you to manage your allocations with just one wallet address.
+
+When you delegate, you transfer 100% of your veOCEAN allocation power for a limited period. After delegation, you cannot manage your allocations until the delegation expires. The delegation expiration date is the same as the veOCEAN lock end date at the time of delegation. If necessary, you can extend your lock before delegating. You can cancel your delegation at any time.
+
+Once delegated, rewards will be sent to the wallet address you delegated to. The delegation receiver is in charge of your rewards and the process of returning those back to you.
+
+To delegate, go to the DF Portal and select the wallet address you wish to delegate to. Click the Delegate button and sign the transaction. You can view information about your delegation in the My Delegations component. If needed, you can cancel delegation to regain your allocation power before the delegation expires.
```
[the remark this answers is not in the document]
Comment (EnzoVezzaro) [reply rewards/veOcean-Data-Farming-Tutorial.md:87]: > Go to the DF Portal and select the wallet address you wish to delegate to. 

I'm a bit confused here. DF Portal you mean homepage or farm page? I imagine Farm page, maybe we can change it to that.

>  You can view information about your delegation in the My Delegations component. 

"component" is a term a developer would use, I'd use instead "in the My Delegations section in Delegate Page."

File: rewards/veOcean-Data-Farming-Tutorial.md [hunk 2]
```diff
@@ -76,5 +74,22 @@ All rewards are paid out in OCEAN tokens. On every Thursday the pink "Claim" but
...
+
+![](./images/veOCEAN-Delegation.png)
```
Comment (idiom-bytes) [inline rewards/veOcean-Data-Farming-Tutorial.md:89]: Image should change base on @trizin latest comments.
Rewards will not be dispensed to the receiver.

File: rewards/images/veOCEAN-Delegation.png
Comment (jamiehewitt15) [inline rewards/images/veOCEAN-Delegation.png:1]: This image could probably be cropped, I don't think we need all the white space on each side

Comment (KatunaNorbert) [reply rewards/images/veOCEAN-Delegation.png:1]: done

**Review by jamiehewitt15 — APPROVED** [review]
Looks good to me

**Pull request merged**